fabric_service_tokens.go

```diff
@@ -0,0 +1,48 @@
+package packngo
+
+import "path"
+
+type FabricServiceTokenType string
+
+const (
+	fabricServiceTokenBasePath                        = "/fabric-service-tokens"
+	FabricServiceTokenASide    FabricServiceTokenType = "a_side"
+	FabricServiceTokenZSide    FabricServiceTokenType = "z_side"
+)
+
+// FabricServiceTokenService interface defines available metro methods
+type FabricServiceTokenService interface {
+	Get(string, *GetOptions) (*FabricServiceToken, *Response, error)
+}
+
+// FabricServiceToken represents an Equinix Metal metro
+type FabricServiceToken struct {
+	ID               string                 `json:"id"`
+	Role             string                 `json:"role,omitempty"`
+	State            string                 `json:"state,omitempty"`
+	MaxAllowedSpeed  int                    `json:"max_allowed_speed,omitempty"`
+	ServiceTokenType FabricServiceTokenType `json:"service_token_type,omitempty"`
+	Connection       *Connection            `json:"interconnection,omitempty"`
+	ConnectionPort   *ConnectionPort        `json:"interconnection_port,omitempty"`
+	Organization     *Organization          `json:"organization,omitempty"`
+}
```

Comment:
Another field to add is `expires_at`: ("2019-09-30T20:22:20", when the token will expire).